tenseal/enc_context.py

```diff
@@ -86,6 +93,7 @@ def __init__(
 
         Args:
             scheme : define the scheme to be used, either SCHEME_TYPE.BFV or SCHEME_TYPE.CKKS.
+            encryption_type : define the encryption type to be used, either ENCRYPTION_TYPE.PUBLIC_KEY, or ENCRYPTION_TYPE.SYMMETRIC.
```

Comment:
Better put the docs in the same order as the parameters (encryption_type after coeff)